app/Config/Database.php

```diff
@@ -43,6 +43,7 @@ class Database extends Config
         'failover'     => [],
         'port'         => 3306,
         'numberNative' => false,
+        'application_name' => '', //supported only in Postgres
```

Comment:
This array is settings for MySQLi. So do not add unrelated config item.
Add in
https://github.com/codeigniter4/CodeIgniter4/blob/902ec98def424032d5fed1f48a3cc4e1efc86c94/app/Config/Database.php#L75-L100